evaluationStage.go

```diff
@@ -458,6 +458,34 @@ func notContainsStage(left interface{}, right interface{}, parameters Parameters
 	return true, nil
 }
 
+func betweenStage(left interface{}, right interface{}, parameters Parameters) (interface{}, error) {
+
+	if left.(float64) <= right.([]interface{})[1].(float64) && left.(float64) >= right.([]interface{})[0].(float64) {
```

Comment:
what if "left/right" is not float64?

Reply:
Have added validation in governor to pass only numeric entities in case of between, so no other value will come.

Reply:
But that's not right. The is a separate library so this should handle the validations.